boards/common/nrf52xxxdk/Makefile.dep

```diff
@@ -1,5 +1,6 @@
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
+  USEMODULE += saul_pwm
```

Comment:
Kconfig change is missing, please re-add ci label when addressed.

Reply:
That CI check is worth its weight in gold. (Metaphorically speaking, assigning some arbitrary weight to the check). Without that, Kconfig migration would be a whack-a-mole. I think I should just put it in my commit hooks.

Reply:
In fact, this would have not been caught by Murdock, since SAUL is not yet fully modeled. Nice catch!